desktop/core/src/desktop/js/reactComponents/FileChooser/PathBrowser/PathBrowser.scss

```diff
@@ -19,82 +19,77 @@
 $icon-width: 24px;
 $icon-height: 24px;
 
-.hue-path-browser {
-  display: flex;
-  width: 100%;
-
-  .hue-path-browser__file-system-icon {
-    object-fit: cover;
-    width: $icon-width;
-    height: $icon-height;
-    margin: 0 vars.$fluidx-spacing-xs;
-  }
-
-  .hue-path-browser__toggle-breadcrumb-input-btn:empty {
-    pointer-events: auto;
-    width: 60px;
-    visibility: visible;
-    border: none;
-    box-shadow: none;
-    padding: 0;
-    background-color: vars.$fluidx-white;
+.antd.cuix {
+  .hue-path-browser {
+    display: flex;
+    width: 100%;
 
-    &:hover {
-      cursor: text;
-      background-color: vars.$fluidx-gray-100;
+    .hue-path-browser__file-system-icon {
+      object-fit: cover;
+      width: $icon-width;
+      height: $icon-height;
+      margin: 0 vars.$fluidx-spacing-xs;
     }
 
-    &:focus {
-      background-color: vars.$fluidx-gray-100;
+    .hue-path-browser__toggle-breadcrumb-input-btn:empty {
+      pointer-events: auto;
+      width: 60px;
+      visibility: visible;
+      background-color: transparent;
+      padding: 0;
+
+      &:hover {
+        cursor: text;
+      }
     }
   }
-}
 
-.hue-path-browser__breadcrumbs {
-  /* stylelint-disable no-invalid-position-at-import-rule */
-  @import './OverflowingItem.scss';
+  .hue-path-browser__breadcrumbs {
+    display: flex;
+    gap: vars.$fluidx-spacing-xs;
+    max-width: 60%;
 
-  display: flex;
-  gap: vars.$fluidx-spacing-xs;
-  max-width: 60%;
+    &:hover {
+      cursor: pointer;
+    }
 
-  &:hover {
-    cursor: pointer;
+    .hue-path-browser__breadcrumb:last-of-type button:last-of-type {
+      color: vars.$fluidx-gray-700;
+    }
   }
 
   .hue-path-browser__dropdown-button {
-    border: none;
-    box-shadow: none;
-    background-color: transparent;
-
     &:hover {
-      background-color: vars.$fluidx-gray-100;
+      background-color: transparent;
     }
 
     &:focus {
-      background-color: vars.$fluidx-gray-100;
+      background-color: transparent;
     }
   }
 
-  .hue-path-browser__breadcrumb:last-of-type button:last-of-type {
-    color: vars.$fluidx-gray-700;
+  .hue-path-browser__breadcrumb-seperator {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 0 0 auto;
   }
-}
 
-.hue-path-browser__breadcrumb-seperator {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex: 0 0 auto;
+  .hue-path-browser__input {
+    width: 300px;
```

Comment:
There are multiple hardcoded widths in this style, do they have some relation or are they completely independent of each other?

Reply:
Independent